python/tests/test_drawing.py

```diff
@@ -1881,56 +1884,58 @@ def test_tree_root_branch(self):
         assert not ("H 0" in snippet2a)  # No root branch
         assert "H 0" in snippet2b
 
+    def verify_known_svg(self, svg, filename, **kwargs):
+        # expected SVG files can be inspected in tests/data/svg/*.svg
+        svg = xml.dom.minidom.parseString(
+            svg
+        ).toprettyxml()  # Prettify for easy viewing
+        self.verify_basic_svg(svg, **kwargs)
+        svg_fn = pathlib.Path(__file__).parent / "data" / "svg" / filename
+        # with open(svg_fn, "wt") as file:  # Uncomment to save new expected files
```

Comment:
This could be implemented as a a pytest flag. No urgent need, maybe next time we touch this code.

Reply:
Good point. I've added this as a comment. Let's see if it sneaks in before automerge.